khronos_api/src/lib.rs

```diff
@@ -34,3 +34,7 @@ pub const WEBGL_IDL: &'static [u8] = include_bytes!("../api_webgl/specs/latest/1
 
 /// The contents of [`webgl2.idl`](https://github.com/KhronosGroup/WebGL/blob/master/specs/latest/2.0/webgl2.idl)
 pub const WEBGL2_IDL: &'static [u8] = include_bytes!("../api_webgl/specs/latest/2.0/webgl2.idl");
+
+/// The contents of the WebGL extension XML files
+/// These are discovered via a build script to avoid having to list each extension by name.
```

Comment:
Minor nit: Probably don't want the second line showing up in the user facing documentation, so we can probably skip this.